tgstation.dme

```diff
@@ -1485,6 +1485,7 @@
 #include "code\modules\client\client_colour.dm"
 #include "code\modules\client\client_defines.dm"
 #include "code\modules\client\client_procs.dm"
+#include "code\modules\client\darkmode.dm"

```

Comment:
TBH I wish our .dme just kinda auto-updated.

Reply:
Update yogbot to do it

Reply:
Or change the mirrorbot code